variables.tf

```diff
@@ -461,3 +461,14 @@ variable "ecs_service_enabled" {
   description = "Whether or not to create the aws_ecs_service resource"
   default     = true
 }
+
+variable "label_order_custom" {
+  type = object({
+    task        = optional(list(string)),
+    service     = optional(list(string)),
+    exec        = optional(list(string)),
+    ecs_service = optional(list(string))
+  })
+  default     = {}
+  description = "Overrides the `labels_order` for the different labels to modify ID elements appear in the `id`"
+}
```

Comment:
This is being different to all other modules that we host on github. Either you have to make them separate variables, like in all other modules or you have to adjust all other modules. In the latter case, i'd prefer if we'd call it `var.label_orders`

Reply:
Having 4 different variables, in that case, seem to be so clean in my mind. I guess that way you always have them together and easy to expand

Reply:
As we've discussed in the meeting, let change this to a `label_orders` object with properties per service, not per label module.